doc/internal/man3/evp_keymgmt_export_to_provider.pod

```diff
@@ -0,0 +1,55 @@
+=pod
+
+=head1 NAME
+
+evp_keymgmt_export_to_provider,
+evp_keymgmt_clear_pkey_cache
+- key material provider export for EVP
+
+=head1 SYNOPSIS
+
+ #include "internal/evp_int.h"
+
+ void *evp_keymgmt_export_to_provider(EVP_PKEY *pk, EVP_KEYMGMT *keymgmt);
+ void evp_keymgmt_clear_pkey_cache(EVP_PKEY *pk);
+
+=head1 DESCRIPTION
+
+evp_keymgmt_export_to_provider() exports the key material from the
+given key I<pk> to a provider via a B<EVP_KEYMGMT> interface, if this
+hasn't already been done.
+It maintains a cache of provider key references in I<pk> to keep track
+of all such exports.
+
+If I<pk> has an assigned internal key, a check is done to see if any
+of its key material has changed since last export.
+If it has (the is_dirty() method returned 1), the cache of already
+exported keys is cleared, and a new export is made with the new key
+material.
+
+evp_keymgmt_clear_pkey_cache() can be used to explicitly clear the
+cache of provider key references.
+
+=head1 RETURN VALUES
+
+evp_keymgmt_export_to_provider() returns a pointer to the newly
+created provider side key, or NULL on error.
+
+=head1 SEE ALSO
+
+L<EVP_PKEY_ASN1_METHOD(3)>
+
+=head1 HISTORY
+
+The functions described here were all added in OpenSSL 3.0.
```

Comment:
I am dubious about the value of the HISTORY section in internal docs. In the public API it is useful to know when a function was introduced if you are wanting to target multiple versions of OpenSSL. For internal functions that isn't the case (you're only ever interested in the "current" version), and even if a function was present in an old release its purpose or signature might have changed over time (which is again not the case with public API functions).

Reply:
Hmmm, good point.  I kinda had that out of habit.  Will remove

Reply:
Done